<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QDialog" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Registeration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: #121212;
    color: #E0E0E0;
    border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>361</width>
     <height>300</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <!-- Username -->
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {
           font-size: 14px;
           font-weight: bold;
           color: white;
       }</string>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtUserName">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
           background-color: #1E1E1E;
           border: 1px solid #333333;
           border-radius: 8px;
           color: #FFFFFF;
           padding: 6px;
       }</string>
      </property>
     </widget>
    </item>

    <!-- Email -->
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
           font-size: 14px;
           font-weight: bold;
           color: white;
       }</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtEmail">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
           background-color: #1E1E1E;
           border: 1px solid #333333;
           border-radius: 8px;
           color: #FFFFFF;
           padding: 6px;
       }</string>
      </property>
     </widget>
    </item>

    <!-- Password -->
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel {
           font-size: 14px;
           font-weight: bold;
           color: white;
       }</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtPassword">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
           background-color: #1E1E1E;
           border: 1px solid #333333;
           border-radius: 8px;
           color: #FFFFFF;
           padding: 6px;
       }</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
    </item>

    <!-- Confirm Password -->
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">QLabel {
           font-size: 14px;
           font-weight: bold;
           color: white;
       }</string>
      </property>
      <property name="text">
       <string>Confirm Password:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtConfPassword">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
           background-color: #1E1E1E;
           border: 1px solid #333333;
           border-radius: 8px;
           color: #FFFFFF;
           padding: 6px;
       }</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
    </item>

    <!-- Nume -->
    <item row="4" column="0">
     <widget class="QLabel" name="label_nume">
      <property name="styleSheet">
       <string notr="true">QLabel {
           font-size: 14px;
           font-weight: bold;
           color: white;
       }</string>
      </property>
      <property name="text">
       <string>Nume:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtNume">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
           background-color: #1E1E1E;
           border: 1px solid #333333;
           border-radius: 8px;
           color: #FFFFFF;
           padding: 6px;
       }</string>
      </property>
     </widget>
    </item>

    <!-- Prenume -->
    <item row="5" column="0">
     <widget class="QLabel" name="label_prenume">
      <property name="styleSheet">
       <string notr="true">QLabel {
           font-size: 14px;
           font-weight: bold;
           color: white;
       }</string>
      </property>
      <property name="text">
       <string>Prenume:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtPrenume">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
           background-color: #1E1E1E;
           border: 1px solid #333333;
           border-radius: 8px;
           color: #FFFFFF;
           padding: 6px;
       }</string>
      </property>
     </widget>
    </item>

    <!-- CNP -->
    <item row="6" column="0">
     <widget class="QLabel" name="label_cnp">
      <property name="styleSheet">
       <string notr="true">QLabel {
           font-size: 14px;
           font-weight: bold;
           color: white;
       }</string>
      </property>
      <property name="text">
       <string>CNP:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txtCNP">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
           background-color: #1E1E1E;
           border: 1px solid #333333;
           border-radius: 8px;
           color: #FFFFFF;
           padding: 6px;
       }</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>

  <!-- Buton Submit -->
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
            background-color: #6200EE;
            color: #FFFFFF;
            border: none;
            border-radius: 8px;
            padding: 5px;
            font-size: 14px;
        }
        QPushButton:hover {
            background-color: #3700B3;
        }
        QPushButton:pressed {
            background-color: #BB86FC;
        }</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
